cdap-app-fabric/src/main/java/co/cask/cdap/gateway/handlers/ProgramLifecycleHttpHandler.java

```diff
@@ -638,7 +641,7 @@ public void getProgramStatusSchedules(HttpRequest request, HttpResponder respond
     final ProgramId triggerProgramId = appId.program(ProgramType.valueOfCategoryName(triggerProgramType),
                                                      triggerProgramName);
     programScheduler.findSchedules(triggerProgramId.toString());
-    Set<ProgramScheduleRecord> schedules = new HashSet<>();
+    Collection<ProgramScheduleRecord> schedules = new HashSet<>();
```

Comment:
so that the result from `filterSchedulesByStatus` method can be assigned to it, which returns a Collection